package.json

```diff
@@ -64,7 +64,7 @@
     "ioredis": "^4.27.6",
     "js-yaml": "^4.1.0",
     "jsonwebtoken": "^8.5.1",
-    "lightning": "^3.3.14",
+		"lightning": "github:alexbosworth/lightning",
```

Comment:
Whitespace + don't want to use main. If it is an unreleased version we should at least pin to a commit hash.